cartographer/mapping/internal/optimization/optimization_problem_3d.cc

```diff
@@ -548,7 +548,11 @@ void OptimizationProblem3D::Solve(
 
       problem.AddResidualBlock(
           SpaCostFunction3D::CreateAutoDiffCostFunction(constraint_pose),
-          nullptr /* loss function */,
+          /* loss function */
```

Comment:
Can we remove this comment? Seems clear, now that we add loss function. It was only there before for readability since `nullptr` could be anything.

Reply:
done